packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts

```diff
@@ -758,7 +759,22 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
         if (freshnessColorEl) {
           freshnessColorEl.style.display = 'none';
         }
-        await context.livenessStreamProvider?.endStream();
+
+        let closureCode = 4000;
```

Comment:
Adding closureCode to `WS_CLOSURE_CODE` enum feels cleaner to me. Perhaps `WS_CLOSURE_CODE.DEFAULT`